Continuous Delivery and Operation Excellence/Continuous Delivery.md

```diff
@@ -0,0 +1,65 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
+
+* Webinar 
+* Self-service content 
+* Office Hours
+* Lab
+
+## Content Delivery Schedule
+
+Plan | Topic | Delivery Format 
+---- | ----- | ---------------
+Week 1 | Fast and Continuous Delivery in SaaS SDLC | Webinar
+Week 2 | Modernizing Compute | Webinar
+Week 3 | Application Storage Modernization | Webinar
+Week 4 | Innovation with Data & AI | Webinar
+Week 5 | Azure Marketplace | Webinar
+Week 6 | Identity in SaaS | Webinar
+Week 7 | The ISV Modernization Journey | Webinar
+Week 8 | Modern Architecture Patterns | Webinar

```

Comment:
These webinars are for the enter program right? should have have this in this file or at the main .md file with reference to devops content?

Reply:
Based on dev's feedback , we should move this as
Week 1 | The ISV Modernization Journey | Webinar
Week 2 | Modern Architecture Patterns | Webinar

Reply:
Agree!